pkgs/applications/editors/emacs/elisp-packages/header-file-mode/default.nix

```diff
@@ -5,13 +5,13 @@
 
 trivialBuild {
   pname = "header-file-mode";
-  version = "unstable-2022-04-19";
+  version = "unstable-2022-05-13";
 
   src = fetchFromGitHub {
```

Comment:
I suppose I could, but the upstream package appears to be unmaintained, so I thought I would take over maintaining it in my fork.

Reply:
Ok, if you intend to keep maintaining it in your fork I think this is fine.
We can always revisit this decision if upstream is revived.

Reply:
Absolutely.  I have opened a PR, but given the repo owner's lack of activity anywhere on GitHub since September 2020, I would be (pleasantly) surprised if it goes anywhere.